From 74c5f6262984a11e028f7bb2da52e965c1949452 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Wed, 26 Sep 2018 18:18:37 +0900
Subject: [PATCH 530/909] arm64: dts: r8a77990: Add renesas id for lvds
 distinction

commit e9d506632abc893f3370a6004743b20be501650f from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Apply this patch to identify LVDS number.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index e01382b7d060..9c077f2bbd54 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -1684,6 +1684,7 @@
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 727>;
 			status = "disabled";
+			renesas,id = <0>;
 
 			ports {
 				#address-cells = <1>;
@@ -1711,6 +1712,7 @@
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 726>;
 			status = "disabled";
+			renesas,id = <1>;
 
 			ports {
 				#address-cells = <1>;
-- 
2.17.1

